AcquisitionInfo/explainer.md

```diff
@@ -0,0 +1,275 @@
+the acquisition information that is recorded for the originally acquired application should be maintained to be the same for all
+other installations.
+
+## Privacy and Security Considerations
+
+1. _Lack of validation_: It is assumed that the install source does not perform any validation
+  or sanitation of the acquisition data retrieved itself.
+
+2. _Lack of user control regarding acquisition identifiers_: This solution provides no method for
+   the user to clear, disable, or modify the data retrieved from `acquisitionInfoProvider.getDetails()`.  We recognize that different vendors may have specific preferences and requirements regarding this aspect,
+   so we defer to the individual install sources to handle that implementation.
+
+3. _Potential misuse of acquisition identifiers_: Acquisition identifiers could potentially be
+   manipulated or obscured by ad networks or referral sources, similar to the existing practices
+  in advertising. However, this behavior is not specific to the proposed API and is already
+  possible with ad campaigns.
+
+This API can only be invoked in a top-level [secure context][secure context]. This is in order to prevent attackers from accessing privacy information that
+is being exposed in this API.
+
+## Glossary
+
+| Term        | Definition                                                          |
+| ----------- | ------------------------------------------------------------------- |
+| Attribution | Identifying the source that led to an installation.                 |
+| Acquisition | Installation of the application in question.                        |
+| PWA         | Progressive Web App                                                 |
+| UWP         | Universal Windows Platform                                          |
+| API         | Application Programming Interface                                   |
+| WinRT       | Windows Runtime                                                     |
+| Store       | Microsoft Store                                                     |
+| App         | Application                                                         |
+| Campaign ID | A unique identifier that is associated with a specific ad campaign. |
+
+[WinRT API]: https://learn.microsoft.com/en-us/uwp/api/windows.services.store.storecollectiondata.campaignid
+[display mode]: https://developer.mozilla.org/en-US/docs/Web/Manifest/display
+[Play Install Referrer API]: https://developer.android.com/google/play/installreferrer
+[Apple Ads Attribution API]: https://developer.apple.com/documentation/ad_services
+[secure context]: https://w3c.github.io/webappsec-secure-contexts/
+[Web Install API]: https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/WebInstall/explainer.md
+
+## Open Questions
+
+### What properties should we be returning for browser installation?
+
+Currently, we are returning the `attributionId` property as the sole descriptor for browser acquisition. Is this enough?
```

Comment:
We should simply specify these as part of the proposal, and then see if others object/have feedback.
Our starting opinion is that we should have the following values:

* HTTP referrer (will be empty for a WebInstall)
* URL at time of installation (will be empty for WebInstall)
* Installation timestamp

Reply:
Do these suggested values add on to the two (`attributionId` and `referral-info`) that you proposed in the previous comment? 
So the full proposed dictionary for browser installations become:
* installSource (This would be the URL at time of installation)
* attributionId
* referralInfo
* httpReferrer
* installBeginTimestampSeconds

I don't see why we need to capture the HTTP referrer if we are already capturing attributionId as the ad campaign and install source as the origin.